BibBuddy/app/src/main/java/de/bibbuddy/Author.java

```diff
@@ -38,7 +40,7 @@ public Author(String firstName, String lastName) {
   }
 
   /**
-   * DUMMY COMMENT - PLEASE ADJUST.
+   * Constructor for optional title.
    *
    * @param firstName test
```

Comment:
```suggestion
   * @param firstName first name of the author of the book
```

Need to adjust all dummy params ("test") - this is just an example